sonar-plugin-api/src/main/java/org/sonar/api/server/ws/internal/SimpleGetRequest.java

```diff
@@ -76,4 +78,16 @@ public SimpleGetRequest setParam(String key, @Nullable String value) {
     return this;
   }
 
+  public Map<String, String> getParams() {
+    return params;
+  }
+
+  @Override
+  public LocalConnector getLocalConnector() {
+    return localConnector;
```

Comment:
either missing a `@CheckForNull` or some check in the implementation is missing to enforce the method's contract